ui/src/pages/H5adDatasetPage.vue

```diff
@@ -0,0 +1,135 @@
+<script setup lang="ts">
+import type {
+  ColDef,
+  GridOptions,
+} from 'ag-grid-enterprise';
+import {
+  ClientSideRowModelModule,
+  MenuModule,
+  ModuleRegistry,
+  RichSelectModule,
+} from 'ag-grid-enterprise';
+
+import { AgGridVue } from 'ag-grid-vue3';
+
+import type { DSH5ad, PlId } from '@platforma-open/milaboratories.samples-and-data.model';
+import type { ImportFileHandle } from '@platforma-sdk/model';
+import type { PlAgHeaderComponentParams } from '@platforma-sdk/ui-vue';
+import { AgGridTheme, makeRowNumberColDef, PlAgCellFile, PlAgColumnHeader } from '@platforma-sdk/ui-vue';
+import { computed } from 'vue';
+import { useApp } from '../app';
+import { agSampleIdColumnDef, getSelectedSamples } from '../util';
+
+ModuleRegistry.registerModules([ClientSideRowModelModule, RichSelectModule, MenuModule]);
+
+type H5adDatasetRow = {
+  readonly sample: PlId;
+  readonly data?: ImportFileHandle | null;
```

Comment:
Do we really need this three state field here? I mean it can be `ImportFileHandle` or `null` or `undefined`.

Reply:
removed undefined state